examples/dataset_exploration/ricci_carrier_transport/explore_carrier_transport.py

```diff
@@ -23,24 +23,29 @@
 from matminer.datasets import load_dataset
 from tqdm import tqdm
 
-from pymatviz import ptable_heatmap, spacegroup_hist
+from pymatviz import count_elements, ptable_heatmap, spacegroup_hist
 from pymatviz.enums import Key
 from pymatviz.io import save_fig
 
 
 # %%
 df_carrier = load_dataset("ricci_boltztrap_mp_tabular")
 
-# getting space group symbols and numbers takes about 2 min
+# TODO: Many structure is a float for some reason?
+df_carrier = df_carrier[
+    df_carrier[Key.structure].apply(lambda x: not isinstance(x, float))
+]
+
```

Comment:
actually, these structures are not floats, they are missing which is interpreted as `float("nan")`. a cleaner way to filter them out would be `df_carrier.dropna(subset=Key.structure)`

Reply:
Thanks for the input. Sorry I didn't check the actual value but the type. I would drop NaN then :)